addon/components/flash-message.js

```diff
@@ -49,7 +49,7 @@ export default Component.extend({
   }),
 
   _setActive: on('didInsertElement', function() {
-    run.next(this, () => {
+    this.pendingSet = run.next(this, () => {
```

Comment:
Yep! When all laid out, it looks a little tougher to read to me though. When nothing should observe an assignment I often forego `set`. But I also understand sticking with the established style in the project if always using `set` is more consistent.  Have a look and let me know if you want to go back to using the `=` assignment:

```javascript
  _setActive: on('didInsertElement', function() {
    set(this, 'pendingSet', run.next(this, () => {
      set(this, 'active', true);
    }));
  })
```